plugin/trino-singlestore/src/test/java/io/trino/plugin/singlestore/TestSingleStoreConnectorTest.java

```diff
@@ -223,6 +223,31 @@ public void testSingleStoreTinyint()
         assertUpdate("DROP TABLE mysql_test_tinyint1");
     }
 
+    // Overridden because the method from BaseConnectorTest fails on one of the assertions, see TODO below
+    @Override
+    public void testInsertIntoNotNullColumn()
```

Comment:
The test comes from https://github.com/trinodb/trino/blob/master/plugin/trino-mariadb/src/test/java/io/trino/plugin/mariadb/BaseMariaDbConnectorTest.java#L265-L289

Reply:
hmm how come it didn't fail when I added it ..

Reply:
SingleStore tests don't run when the PR comes from forked repository because it requires a license. 
https://github.com/trinodb/trino/blob/master/plugin/trino-singlestore/src/test/java/io/trino/plugin/singlestore/TestingSingleStoreServer.java#L31

Reply:
oh right, is SingleStore the only one that failed because of my change?

Reply:
I believe so. This master job https://github.com/trinodb/trino/runs/7822744342 was green except for `test (plugin/trino-singlestore)` yesterday.